src/main/resources/reference.conf

```diff
@@ -262,6 +259,15 @@ resourceTypes = {
       delete_persistent_disk = {
         description = "delete persistent disk in this billing-project"
       }
+      list_kubernetes_app = {
+        description = "list all kubernetes applications in this billing-project"
+      }
+      create_kubernetes_app = {
+        description = "create kubernetes application in this billing-project"
+      }
+      delete_kubernetes_app = {
+        description = "delete kubernetes application in this billing-project"
```

Comment:
this is delete _any_ k8s app in the billing-project, correct? But the billing-project owner won't have delete access on the kubernetes-app resource so they won't be able to remove it from sam.